examples/reference/custom_components/PreactComponent.md

```diff
@@ -0,0 +1,401 @@
+# `PreactComponent`
```

Comment:
When the `_esm` script is not valid I get exception about `tsx` problem. That is a bit surpricing as I'm not using React. Should sucrase be used with the `PreactComponent`?

Reply:
Yes we still have to compile some of the other ESM based features but I've now scoped this better. Also compile errors are rendered in the browser if `--autoreload` is enabled.